socorro/external/es/supersearch.py

```diff
@@ -422,6 +432,28 @@ def get(self, **kwargs):
 
             search.aggs.bucket('signature', sig_bucket)
 
+        # Create date histograms.
+        if params.get('_histogram.date'):
```

Comment:
You can aggregate on all fields in the same request (which is cool, because that means you can build many things with just one request to ES :) ).